capstone-app/src/app/modules/maps/map/map.component.spec.ts

```diff
@@ -1,20 +1,82 @@
+/// <reference types="googlemaps" />
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { MapComponent } from './map.component';
+import { GoogleMapsModule } from '@angular/google-maps';
+import { By } from '@angular/platform-browser';
+
 
 describe('MapComponent', () => {
   let component: MapComponent;
   let fixture: ComponentFixture<MapComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ MapComponent ]
+      declarations: [ MapComponent ],
+      imports: [ GoogleMapsModule ]
     })
     .compileComponents();
   }));
 
   beforeEach(() => {
     fixture = TestBed.createComponent(MapComponent);
-    component = fixture.componentInstance;
     fixture.detectChanges();
   });
+
+  it('should create the app', () => {
+    // const fixture = TestBed.createComponent(MapComponent);
+    // const app = fixture.componentInstance;
+    expect(fixture.componentInstance).toBeTruthy();
+  });
+
+  it('test map marker click event', () => {
+    const fixture = TestBed.createComponent(MapComponent);
+    fixture.detectChanges();
+
+    const coordinates = new google.maps.LatLng({lat: 26.011760, lng:  -80.139050});
+
+    fixture.componentInstance.markers.push(coordinates);
+    fixture.detectChanges();
+
+    const markerDebug = fixture.debugElement.query(By.css('.marker'));
+    const markerComponent = markerDebug.componentInstance;
+
+    expect(markerComponent.getPosition()).toEqual(coordinates);
+  });
```

Comment:
I see that you have a test for one marker and one for multiple markers. The behavior of the single marker appears to be effectively the same as the multiple marker. No special handling is needed for the single marker case, so let's remove that test and only keep the multiple marker test.